sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -267,7 +273,7 @@ object PROCESS_TABLES extends QueryTest with SQLTestUtils {
       case NonFatal(_) => Nil
     }
     versions
-      .filter(v => v.startsWith("3") || !TestUtils.isPythonVersionAtLeast38())
+      .filter(v => v.startsWith("3") && isPythonVersionAtLeast37)
       .filter(v => v.startsWith("3") || !SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9))
```

Comment:
`.filter(v => v.startsWith("3") || !SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9))` 

After thinking about it, I will delete this condition because it will not filter out any data